components/panel-body/style.scss

```diff
@@ -0,0 +1,4 @@
+.components-panel-body {
+	padding: 15px;
+	box-shadow: 0 0 0 1px $light-gray-500;
```

Comment:
A box-shadow is more "generic" because we could use several panels one after another and having only one line (the borders collapse) while this is not the case with border.